packages/profiling-browser/src/hubextensions.ts

```diff
@@ -0,0 +1,254 @@
+import { WINDOW } from '@sentry/browser';
```

Comment:
exported code here is responsible for providing both the implementation of the patched startTransaction functionality (wrapTransactionWithProfiling) and the code that performs the patching addExtensionMethods